packages/runtime/src/widgets/Charts/index.tsx

```diff
@@ -59,13 +70,24 @@ const tabsConfig = {
 };
 
 export const Chart: React.FC<ChartProps> = (props) => {
-  const { type } = props;
+  const context = useScreenContext();
+
+  const config = useMemo(
+    () =>
+      evaluate(
+        context as ScreenContextDefinition,
+        props?.config?.toString()?.replace(/['"]\$\{([^}]*)\}['"]/g, "$1"),
+      ) as ChartConfigs,
+    [props.config, context],
+  );
 
-  if (!type) {
-    return <b>Chart type missing</b>;
-  }
+  if (!config) return null;
```

Comment:
We can return an Error widget or something to let them know the config is bad/missing. See the runtime render method for an example.